blast/2.15.0/Dockerfile

```
@@ -0,0 +1,85 @@
+FROM ncbi/edirect:20.6 as edirect
+FROM ubuntu:22.04
+ARG blast_version
+ARG vdb_version
+ARG num_procs=8
+LABEL Description="NCBI BLAST" Vendor="NCBI/NLM/NIH" Version=${blast_version} Maintainer=camacho@ncbi.nlm.nih.gov
+
+USER root
+WORKDIR /root/
+RUN mkdir -p /blast/bin /blast/lib
+
+RUN apt-get -y -m update && apt-get install -y build-essential curl libidn12 libnet-perl perl-doc liblmdb-dev wget libsqlite3-dev \
```

```suggestion
RUN apt-get -y -m update && apt-get install -y build-essential curl libidn12 libnet-perl perl-doc liblmdb-dev wget libsqlite3-dev cmake \
```
In ubuntu:22.04 you can use `cmake` installed by `apt-get`.